ChangeLog.d/md_info_from_ctx.txt

```diff
@@ -1,3 +1,3 @@
 Features
-   * Add a function to extract message digest information from a message
-     digest context.
+   * The new function mbedtls_md_info_from_ctx() recalls the message digest
+     information that was used to set up a message digest context.
```

Comment:
Why change from the the "Add function foo()" style?

Reply:
As stated before, "the new function" doesn't sound to me like the function was added, it rather states that this new function **now** `recalls the message digest [...]`.
Subjective: `Add _function_name()_ to extract` > `Add function _function_name()_` > `The new function`.

Reply:
On reflection I personally prefer “the new function …”, but that's a matter of preference, so I've changed this back.

Reply:
Re: "Add ..." vs "The new ..." is a case of active vs passive, us saying what we have done (in present tense, as is common for release notes) vs documenting what is done (it's quite common for documentation to be passive voice).